src/backend/distributed/commands/create_distributed_table.c

```diff
@@ -104,6 +104,7 @@ master_create_distributed_table(PG_FUNCTION_ARGS)
 	char distributionMethod = LookupDistributionMethod(distributionMethodOid);
 
 	EnsureSchemaNode();
+	EnsureAllNodesActivated();
```

Comment:
That's right, it is actually not necessary. But, I kept it to make it consistent `create_distributed_table()` and `create_reference_table()`. I'm slightly inclined to keep it as it is. But, could change as well.